pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1337,11 +1345,20 @@ def create_model(df_sets, df_parameters, default={}):
         )
     model.p_epsilon_Treatment = Param(
         model.s_R,
-        model.s_B,
+        model.s_WT,
         default=1.0,
         initialize=model.df_parameters["TreatmentEfficiency"],
         doc="Treatment efficiency [%]",
     )
+
+    model.p_epsilon_Treatment_Removal = Param(
+        model.s_R,
+        model.s_WT,
+        model.s_QC,
+        default=1.0,
```

Comment:
A minor point, but I think I'd recommend using 0 as the default parameter value for treatment removal.

Reply:
One more minor point - by our naming conventions, this variable should be model.p_epsilon_TreatmentRemoval (no underscore between Treatment and Removal). Can this also be updated? (e.g. model.p_sigma_NetworkNode)

Reply:
Corrected these!